dbt/models/marts/sections/dim_section_instructors.sql

```diff
@@ -0,0 +1,46 @@
+-- model: dim_section_instructors
+-- scope: all teachers who created sections since 10/17/2023 when the co-teacher feature was added to the platform. 
+
+-- option 1: dim_section_instructors
+-- this is BP in terms of bringing in this data. we would then affect dim_teachers using this dim model.
+
+
+with 
+section_instructors as (
+    select *, 
+        case when invited_by_id is null -- wasn't invited (is the teacher)
+            then 1 else 0 
+        end as is_section_owner
+    from {{ ref('stg_dashboard__section_instructors') }}
+),
+
+sections as (
+    select *
+    from {{ ref('dim_sections') }}
+),
+
+combined as (
+    select 
+        -- coteacher
+        section_instructors.instructor_id       as teacher_id,
+        section_instructors.status,
+        section_instructors.invited_by_id       as invited_by_teacher_id,
+        section_instructors.is_section_owner,
```

Comment:
```suggestion
 --       section_instructors.is_section_owner,
 case when section_instructors.instructor_id = sections.teacher_id then 1 else 0 is_section_owner
```
A couple of weeks ago I ran into some use cases that highlighted this is the way to identify section owners. Some explanation of the use cases in the threads linked [here ](https://codedotorg.slack.com/archives/C05B9AWTTD5/p1708555911819539)